src/parser/unpyc.cc

```diff
@@ -203,18 +377,79 @@ void parseCode(dbif::ObjectHandle code) {
   if (!bytecodeField)
     return;
   auto bytecodeBlob = makeSubBlob(code, "code", bytecodeField.raw_value);
-  parseLnotab(code, bytecodeBlob);
+  auto lines = parseLnotab(code, bytecodeBlob);
+  StreamParser parser(bytecodeBlob, 0);
+  while (!parser.eof()) {
+    uint64_t arg = 0;
+    int arg_width = 0;
+    parser.startChunk("pyc_insn", QString("insn_%1").arg(parser.pos()));
+restart:
+    uint8_t opcode = parser.getByte("opcode");
+    bool found = false;
+    for (auto &op : pyc_ops) {
+      if (op.opcode == opcode && (op.flags & version.flags) == op.flags) {
+        if (version.flags & PYVER_FLAG_WORDCODE) {
+          arg <<= 8;
+          uint8_t cur_arg = parser.getByte("arg");
+          arg |= cur_arg;
+          arg_width += 8;
+        } else {
+          if (op.type != PYC_OP_SIMPLE) {
+            arg <<= 16;
+            auto cur_arg = parser.getLe16("arg", 1);
+            if (cur_arg.size()) {
+              arg |= cur_arg[0];
+              arg_width += 16;
+            }
+          }
+        }
+        switch (op.type) {
+          case PYC_OP_EXT:
+            goto restart;
+          case PYC_OP_SIMPLE:
+            parser.setComment(op.name);
+            break;
+          case PYC_OP_JREL:
+            arg += parser.pos();
+            // fallthru
+          case PYC_OP_NUM:
+          default:
+            parser.setComment(QString("%1 %2").arg(op.name, QString("%1").arg(arg)));
+            break;
+        }
+        parser.endChunk();
+        found = true;
+        break;
+      }
+    }
+    if (!found)
+      return;
+  }
+#if 0
+  for (auto &line : lines) {
+    // XXX set some sort of a prop with line no
+    bytecodeBlob->syncRunMethod<dbif::ChunkCreateRequest>(
+      QString("line_%1").arg(std::get<2>(line)), "pyc_line_tag", dbif::ObjectHandle(), std::get<0>(line), std::get<1>(line));
+  }
+#endif
```

Comment:
What's the reason you decided to not include any information about line number in code blob ?

BTW. I didn't know github displays always false if-defs as comments - neat.